### PR TITLE
Feature/gg 29 create new employee face id and database

### PR DESCRIPTION
**Pull request by curlyjay**
Saves photos from faceID code to database.
Add and remove employee facial data and web interface data.
Get and add employee photos to database and local directories.
Update with GG-60 feature; saves photo to database and local directories without having to make copies.
Update txt file write and read.

### CODE REVIEW

**Comment by lb-cake** [comment]
After running make migrations, migrate, and finally run, I get the following error.

Screen Shot 2019-07-22 at 2 29 58 PM

Am I doing something wrong?

**Comment by curlyjay** [comment]
Maybe try running 'python3 manage.py showmigrations' before 'python3 manage.py makemigrations'

Are you trying to flush the database out?

**Comment by kemptonk** [comment]
@parksWill I'm not getting the same error. After researching, it looks like you just need to make sure things are migrated properly? So I would try resetting the migrations and flushing the database.

File: FaceID/DB_faceID.py [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+
+
+def token():
+
+    # Enter the admin's password
+    # (getpass is used so the password doesn't have to be hard-coded into the program)
+    password = getpass.getpass(prompt="Admin Password: ")
+    # Create the admin data
+    data = {
+        "username": "jay",
```
Comment (parksWill) [inline FaceID/DB_faceID.py]: can we use any other username besides the string "jay"? For instance, "admin"?

**Comment by curlyjay** [comment]
Yes! I'll change it and push the edit

________________________________

**Review by lb-cake — APPROVED** [review]

**Pull request merged**